notebooks/quickstart.ipynb

```diff
@@ -120,18 +122,115 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "--------------------------------\n",
+    "## The basics of `spaghetti`\n",
+    "### 1. Creating a network instance\n",
+    "\n",
+    "Within `spaghetti` network objects can be created from [a variety of objects](https://pysal.org/spaghetti/generated/spaghetti.Network.html#spaghetti-network), the most common being shapefiles (read in as file paths) and `geopandas.GeoDataFrame` objects. However, a network could also be created from `libpysal` geometries, as demonstrated in the [connected components tutorial](https://pysal.org/spaghetti/notebooks/connected-components.html#1.-Instantiate-a-network-from-two-collections-of-libpysal.cg.Chain-objects) or a simply as follows:\n",
+    "\n",
+    "```python\n",
+    "from libpysal.cg import Point, Chain\n",
+    "import spaghetti\n",
+    "# create the network\n",
+    "ntw = spaghetti.Network(in_data=Chain([Point([1, 1]), Point([2, 1])]))\n",
+    "```\n",
+    "\n",
+    "This will create a single-segment network.\n",
+    "\n",
+    "The stucture and characterstics of the networks are topics of research in many areas. However, networks are also utilized as the study space in many applications. In these cases the actual objects of interest are network-based events.\n",
```

Comment:
Integrate for sentence 1:
"""
The stucture and characterstics of the networks, can be quantitatively described with `spaghetti` and are topics of research in many areas. However, networks are also utilized as the study space, containing observations or events of interest, in many applications. In these cases the actual objects of interest that will be analysed in the geographic space a network provides, are "network-based events".
""""

Please check for correctness